src/incremental/compareAST.ml

```diff
@@ -3,212 +3,284 @@ open CilMaps
 
 module StringMap = Map.Make(String)
 
-type method_rename_assumption = {original_method_name: string; new_method_name: string; parameter_renames: string StringMap.t}
-type method_rename_assumptions = method_rename_assumption VarinfoMap.t
+type method_rename_assumptions = varinfo VarinfoMap.t
+type glob_var_rename_assumptions = varinfo VarinfoMap.t
```

Comment:
What's the meaning of these maps now? An assumption would be that the key `varinfo` is renamed to the value `varinfo`? Might be worth adding a comment about it.